README.md

```diff
@@ -211,6 +212,16 @@ oauth2Client.setCredentials({
 
 Once the client has a refresh token, access tokens will be acquired and refreshed automatically in the next call to the API.
 
+Refresh tokens may stop working after they are granted, either because:
```

Comment:
@bshaffer thinking out loud - is there a way a user can hand us a refresh token, and we can tell them if it's valid or not?  Bonus points if we can tell them *why* it isn't valid.  I know [this fella](https://developers.google.com/identity/sign-in/web/backend-auth#calling-the-tokeninfo-endpoint) exists for id tokens, but I have no idea if such a thing could/would work for a refresh token.

Reply:
I do not believe there is any way to tell WHY a refresh token is invalid. There is a oauth "token info" endpoint (https://www.googleapis.com/oauth2/v3/tokeninfo) but it does not accept refresh tokens. 

...and the only way I'm aware of to tell IF it's valid is to try using it to fetch an access token.